src/utils/formatCoffeeScriptAst.ts

```diff
@@ -70,7 +72,7 @@ function shouldTraverse(value) {
  * CoffeeScript AST nodes are always instances of a custom class, so use the
  * constructor name to distinguish between node children and non-node children.
  */
-function isNode(value) {
+function isNode(value: {}): boolean {
```

Comment:
Perhaps `any` would be good. I think the right return type may actually be `value is Node` rather than `boolean`, as that provides TypeScript with the ability to refine types using `isNode` calls. For example, it would know that in this `if` block that `n` is a `Node`:

```ts
If (isNode(n)) {
  // do something with `n` that assumes it’s a `Node`
}
```